SketchUp/Sketchup/FrameChangeObserver.rb

```diff
@@ -61,33 +71,37 @@ class Sketchup::FrameChangeObserver
   #
   # @example
   #   class MyFrameChangeObserver
-  #     def frameChange(from_page, to_page, percent_done)
-  #       puts "From page #{from_page.name} to #{to_page.name} (#{percent_done * 100}%)"
+  #     def frameChange(from_scene, to_scene, percent_done)
+  #       if percent_done == 0.0
+  #         puts "Animating to scene '#{to_scene.name}':"
+  #       else
+  #         puts format("% 7.2f %",percent_done*100)
+  #       end
   #     end
   #   end
-  #   id = Sketchup::Pages.add_frame_change_observer(MyFrameChangeObserver.new)
+  #   @id = Sketchup::Pages.add_frame_change_observer(MyFrameChangeObserver.new)
   #
   # @note The +from_scene+ argument into this callback does not appear to
   #   be populated on the PC. You can store a variable that keeps track of
   #   the +to_scene+ and then use that on a subsequent Scene selection to
   #   determine the last Page that the user was on.
   #
-  # @param [Float] percent_done
-  #   The percentage of transition between
-  #   the two scene pages.
+  # @param from_scene [Sketchup::Page, nil]
```

Comment:
For now I'll just change the few `@params` back in these 2 files, and we can worry about this until later on.

The thought occurs that YARD can read the stubs, and use a modified stubs template to output stubs with params reordered.